sekoia_automation/account_validator.py

```diff
@@ -0,0 +1,37 @@
+from abc import abstractmethod
+
+import requests
+
+from sekoia_automation.module import ModuleItem
+
+
+class AccountValidator(ModuleItem):
+    VALIDATION_CALLBACK_URL_FILE_NAME = "validation_callback_url"
+
+    @abstractmethod
+    def validator(self) -> bool:
+        """To define in subclasses. Validates the configuration of the module.
+
+        Returns:
+            bool: True if the module is valid, False otherwise
+        """
+
+    def execute(self):
+        """Validates the account (module_configuration) of the module
+        and sends the result to Symphony."""
+        # Call the actual validation procedure
+        status = self.validator()
```

Comment:
Before running the validation we must retrieve the secrets from the API the same way the actions do it.

Reply:
I don't get it, why do we need to do that?

Reply:
The secrets are not passed to the action worker so we need a way to retrieve them.